flocker/control/_persistence.py

```diff
@@ -173,7 +173,7 @@ def default(self, obj):
             result[_CLASS_MARKER] = obj.__class__.__name__
             return result
         elif isinstance(obj, PClass):
-            result = obj.evolver().data
```

Comment:
:question: Create an upstream issue about the usefulness of this being a public method and link to it here.